apps/airtable-cache/src/lib/cache.ts

```diff
@@ -0,0 +1,74 @@
+import z from 'zod';
+import { sql } from 'kysely';
+import { db } from '../db/client';
```

Comment:
I reckon we should probably scrap the entire database, and just use an in-memory cache.

Postgres uses up a bunch of memory in practice (~200MB) and we probably don't need all its functionality.

Maybe we want to keep Postgres if we want to set up webhooks, so we don't set up webhooks then forget about them.